packages/functions/src/emojiList.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Generate a formatted list of items for Discord using emojis
+ * @param items - An array of strings to format
+ * @param doFirst - Whether the first line should have an list starting emoji (true) or be left blank to use as the title (false)
```

Comment:
`doFirst` is very undescriptive as a variable name even when we have JSDoc. Switch to `useEmojiForFirstLine` even though it is longer, it says what it does.